docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -183,6 +183,14 @@ sometimes be started twice.
 If you do specify `.spec.parallelism` and `.spec.completions` both greater than 1, then there may be
 multiple pods running at once.  Therefore, your pods must also be tolerant of concurrency.
 
+### Pod Backoff failure policy
+
+There are situations where one wants to fail a Job after some amount of retries due to a logical error in configuration etc.
+To do so `.spec.template.spec.backoffLimit` can be set to specify the number of retries before considering a Job as failed.
+`.spec.template.spec.backoffLimit` is set by default to 6. Failed Pods associated to the Job are recreated by the Job controller
+with an exponential back-off delay (10s, 20s, 40s ...) capped at six minutes, and is reset if no new failed Pods appear before
```

Comment:
Remove the `capped at six minutes` since we cap at the amount set in `backoffLimit`.

Reply:
Discussed on slack, this is true (https://github.com/kubernetes/kubernetes/pull/51153/commits/1dbef2f113b3be843923cd4e7fdd180b332d86be#diff-3fc7e327f1d875253cb1603875cd1d4fR107) so no need to change it.